app/services/plugin_service.py

```
@@ -0,0 +1,61 @@
+from app.db.supabase_client import supabase
+from app.models.plugin import (
+    PluginConfigurationCreate,
+    PluginConfigurationUpdate,
+)
+from app.services.key_service import KeyService
+from typing import List, Optional
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)